.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,78 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 🐞 Bug Report
+description: File a bug report
+title: "Brief summary of what bug or error was observed"
+labels: ["type: bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for stopping by to let us know something could be better!<br/>
+        ### PLEASE READ PRIOR TO CREATING BUG
+        If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing it here on GitHub. This will ensure a timely response.
+        
+        Please run down the following list and make sure you've tried the usual "quick fixes":
+        - Search the [current open issues](https://github.com/GoogleCloudPlatform/cloud-sql-go-connector/issues)
+        - Check for answers on [StackOverflow](https://stackoverflow.com/questions/tagged/google-cloud-sql) (under the 'google-cloud-sql' tag)
+        
+        If you are still having issues, please include as much information as possible below! :smile:
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Bug Description
+      description: "Please enter a detailed description of the bug, and any information about what behavior you noticed and why it is defective or unintentional."
+    validations:
+      required: true
+  - type: textarea
+    id: example-code
+    attributes:
+      label: Example code (or command)
+      description: "Please paste any useful application code related to the bug below. (if your code is in a public repo, feel free to paste a link!)"
+      render: Go
```

Comment:
@enocom I wonder if it is best to have these not render as code but instead be regular markdown so that users can respond with test and code or multiple code blocks? Thoughts?

Reply:
See `Example Code` section of [Python Connector bug](https://github.com/GoogleCloudPlatform/cloud-sql-python-connector/issues/new?assignees=&labels=type%3A+bug&template=bug-report.yaml&title=Brief+summary+of+what+bug+or+error+was+observed) for context

Reply:
Let's use the same approach whatever we do. I'm sure people will misuse it either way.